sections/dir.zsh

```diff
@@ -21,7 +21,7 @@ SPACESHIP_DIR_COLOR="${SPACESHIP_DIR_COLOR="cyan"}"
 spaceship_dir() {
   [[ $SPACESHIP_DIR_SHOW == false ]] && return
 
-  local dir
+  local 'dir'
```

Comment:
Please leave a comment about using quotes here.

Reply:
This should be default behavior for all unassigned local variables. Documenting it in `doc` would be better than leaving a single comment.

- http://mywiki.wooledge.org/Quotes
- http://zsh.sourceforge.net/Doc/Release/Shell-Grammar.html#Aliasing